doc/manual/manual.tex

```diff
@@ -5945,36 +5945,33 @@ \subsubsection{Artificial viscosity smoothing}
-with the displacement $\mathbf d$ the difference between a point's
-position at time $t$ and at time zero.}
+\note{In this section, following \cite{Becker2003, dahlen1998theoretical}, we denote the velocity gradient tensor as $\mathbf G$, where $\mathbf G = \nabla \mathbf u^T$,
+and $\mathbf u$ is the velocity. Note that this is different from the definition of the strain rate $\epsilon(\mathbf u)$, which only contains the symmetric part of $\mathbf G$. We then denote the deformation gradient (or deformation) tensor by $\mathbf F$, where $\mathbf F$ is the tensor that
+deforms an initial state $\mathbf x$ into an deformed state $\mathbf r = \mathbf F \mathbf x$.}
 
 In many geophysical settings, material properties, and in particular the rheology, do not only depend
 on the current temperature, pressure and strain rate, but also on the history of the system. 
 This can be incorporated in \aspect{} models by tracking history variables through compositional fields. 
-In this cookbook, we will show how to do this by tracking the strain that accumulates over time at every 
+In this cookbook, we will show how to do this by tracking the strain that idealized little grains of finite size accumulate over time at every 
 (Lagrangian) point in the model.
 
-Here, we use a material model plugin that defines the compositional fields as the components of the strain 
-tensor $e_{ij}$, and modifies the right-hand side of 
+Here, we use a material model plugin that defines the compositional fields as the components of the deformation 
+gradient tensor $\mathbf F_{ij}$, and modifies the right-hand side of 
 the corresponding advection equations to accumulate strain over time. This is done by adjusting the 
 \verb!out.reaction_terms! variable:
 \lstinputlisting[language=C++]{cookbooks/finite_strain/finite_strain.cc}
 
-Let us denote the accumulated strain at time step $n$ as $e^n$. We can express it 
-as the sum of the strain $e^{n-1}$ at the previous time step, rotated by the rotational 
-component of the velocity field, plus the strain increment $\varepsilon(\mathbf u^n) \Delta t^n$ 
-accumulated over the current time step.
-Hence, the right-hand side term of the advection equation for the accumulated strain consists of two parts:
-The first one, $R e^{n-1} R^T$, rotates $e^{n-1}$, the accumulated strain from all the previous time steps;
-and the second part adds the strain of the current time step. 
+Let us denote the accumulated deformation at time step $n$ as $\mathbf F^n$. We can calculate its time derivative  
+as the product of two tensors, namely the current velocity gradient $\mathbf G_{ij} = \frac{\partial u_i}{\partial x_j}$ and the deformation gradient $\mathbf F^{n-1}$ accumulated up to the previous time step, in other words $\frac{\partial \mathbf F}{\partial t} = \mathbf G \mathbf F$, and $\mathbf F_0 = \mathbf I$, with $\mathbf I$ being the identity tensor.
```

Comment:
I think you will want to call it `F^0` since you've had the time step as an upper index above.